### PR TITLE
Import strings from android-l10n

### PR DESCRIPTION
**Pull request by Pike**
State: mozilla-l10n/android-l10n@4c73402212e4766cf050b520318cb56961c78e2b

### CODE REVIEW

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -169,6 +169,10 @@
          changes the app and the language of the speech-recognition-based search to 'Italian'. -->
     <string name="settings_language_italian">義大利語</string>
 
+    <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
+     changes the app and the language of the speech-recognition-based search to 'Italian'. -->
```
Comment (MortimerGoro) [inline app/src/main/res/values-zh-rTW/strings.xml]: Wrong comment: Italian

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -169,6 +169,10 @@
          changes the app and the language of the speech-recognition-based search to 'Italian'. -->
     <string name="settings_language_italian">Italiano</string>
 
+    <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
+     changes the app and the language of the speech-recognition-based search to 'Italian'. -->
```
Comment (MortimerGoro) [inline app/src/main/res/values-it/strings.xml]: Wring comment: Italian

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -169,6 +169,10 @@
          changes the app and the language of the speech-recognition-based search to 'Italian'. -->
     <string name="settings_language_italian">Italiano</string>
 
+    <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
+     changes the app and the language of the speech-recognition-based search to 'Italian'. -->
```
Comment (MortimerGoro) [inline app/src/main/res/values-es/strings.xml]: Wring comment: Italian

File: app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -169,6 +169,10 @@
          changes the app and the language of the speech-recognition-based search to 'Italian'. -->
     <string name="settings_language_italian">Italien</string>
 
+    <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
+     changes the app and the language of the speech-recognition-based search to 'Italian'. -->
```
Comment (MortimerGoro) [inline app/src/main/res/values-fr/strings.xml]: Wring comment: Italian

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -169,6 +169,10 @@
          changes the app and the language of the speech-recognition-based search to 'Italian'. -->
     <string name="settings_language_italian">Italienisch</string>
 
+    <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
+     changes the app and the language of the speech-recognition-based search to 'Italian'. -->
+    <string name="settings_language_russian">Russisch</string>
```
Comment (MortimerGoro) [inline app/src/main/res/values-de/strings.xml]: Wring comment: Italian

**Comment by Pike** [comment]
These comments are copied over from the en-US file, which had that bug. They'll be fixed next time a language changes a string in FxR.

**Comment by MortimerGoro** [comment]
Perfect, thanks

**Pull request merged**